legend-engine-xt-nonrelationalStore-mongodb-executionPlan-test/src/main/resources/core_mongodb_execution_test/test_setup/testSetupDSL.txt

```diff
@@ -89,7 +89,7 @@ MongoDBConnection test::mongodb::testConnection
 {
     database: userDatabase;
     store: test::mongodb::mydatabase;
-    serverURLs: [localhost:__PORT_TOKEN__];
+    serverURLs: [__HOST_TOKEN__:__PORT_TOKEN__];
```

Comment:
All this works from connectivity perspective, but when the hostname have characters like '-', the parser fails.  This is a limitation on what kind of hostname you will be supporting, and one that we should fill the gap to fix some of the e2e testing we do.

Reply:
Fixed the parser to accept - as well, but we might have to support a wider set of values potentially.